src/concurrency/timestamp_ordering_transaction_manager.cpp

```diff
@@ -104,13 +108,18 @@ Transaction *TimestampOrderingTransactionManager::BeginTransaction() {
 
 void TimestampOrderingTransactionManager::EndTransaction(Transaction *current_txn) {
   EpochManagerFactory::GetInstance().ExitEpoch(current_txn->GetEpochId());
+  auto &log_manager = logging::LogManager::GetInstance();
```

Comment:
Is EndTransaction only for read only transactions? and if so, do we need to log anything? 
I think we can just call `log_manager.DoneLogging()`